dts/bindings/power-domain/power-domain-gpio.yaml

```diff
@@ -30,3 +30,13 @@ properties:
     required: false
     default: 0
     description: Off delay time, in microseconds
+
+  init-priority:
```

Comment:
for me it's a -1 to set init priorities in DT, this has nothing to do with hardware description. We should finally fix the init priority mess in Zephyr.

Reply:
I agree.

Reply:
Agreed as well!